resources/views/domestic-address-search-bar-javascript.blade.php

```diff
@@ -80,9 +80,9 @@ class="mt-2 px-2 py-1 bg-white text-green-500 border-2 border-green-400 rounded-
     </form>
 </div>
 
-<script async src="https://maps.googleapis.com/maps/api/js?key={{ config('addresses.google_api_key') }}&libraries=places&callback=initMap">
+<script async src="https://maps.googleapis.com/maps/api/js?key={{ config('google-api.places.key') }}&libraries=places&callback=initMap">
```

Comment:
Another example of something that should probably be pushed to a stack in the base layout to get this script rendered in the proper HTML location.